theme/secondary-menu/_base.scss

```diff
@@ -7,8 +7,9 @@
 }
 
 #wb-sec {
-	margin-top: 20px;
```

Comment:
I replaced ``margin-top`` with ``padding-top`` to allow that space to retain the left nav's background colour. It was part of my attempt to keep the [WET theme's secondary menu demo](https://wet-boew.github.io/v4.0-ci/theme/content-secmenu-en.html) looking the same as before.

Upon further observation, this PR doesn't seem to have turned out as well as I'd hoped. In the WET 4.0.22 screenshot below, the bottom of the left nav is light grey. But in both PR screenshots, it's white.

Screenshot of WET 4.0.22 (hasn't changed between 4.0.0-4.0.24):
![wet-4 0 22-theme_secmenu_margin-top-20](https://cloud.githubusercontent.com/assets/1907279/23409651/59573652-fd9a-11e6-9ddf-a8727ca83655.png)

Screenshot of this PR with ``margin-top`` (white background above the menu and white space below it):
![wet-4 0 25pr-theme_secmenu_margin-top-20](https://cloud.githubusercontent.com/assets/1907279/23409580/134e5ea6-fd9a-11e6-96d2-30c52e69a5a1.png)

Screenshot of this PR with ``padding-top`` (white space below the menu):
![wet-4 0 25pr-theme_secmenu_padding-top-20](https://cloud.githubusercontent.com/assets/1907279/23409589/1a4adf72-fd9a-11e6-91d2-9c3d2dfa4669.png)